src/ios/removeFromStaticLibraries.js

```diff
@@ -14,6 +15,8 @@ module.exports = function removeFromStaticLibraries(project, path, opts) {
   project.removeFromPbxBuildFileSection(file);
   project.removeFromPbxFrameworksBuildPhase(file);
   project.removeFromLibrarySearchPaths(file);
+  project.removeFromProductsPbxGroup(file);  // @todo this does not work but it should
```

Comment:
Fix this, cover by tests (optionally, if that's a hotfix) and we're ready to go 👍